core/src/poh_recorder.rs

```diff
@@ -139,7 +142,7 @@ impl PohRecorder {
 
         if let Some(target_tick) = self.start_leader_at_tick {
             // we've reached target_tick OR poh was reset to run immediately
-            if next_tick >= target_tick || self.start_tick + self.grace_ticks == target_tick {
+            if next_tick >= target_tick || self.start_tick + self.grace_ticks >= target_tick {
```

Comment:
why did you change this line?

Reply:
:) I had the opposite question in this PR https://github.com/solana-labs/solana/pull/5127

Reply:
are we sorted, then?

Reply:
Yes. Sounds good.